### ISSUE TITLE
AM2 / thaumcraft

### ISSUE THREAD

**Issue by VxeR** [DrasticDemise/Hubris#28]
Besides problem with recall stones (different issue) I have a problem with Ars Magica 2 quest and those taintacles.

AM2: I can't help it but I have done 40/40 quests and yet I am at 97% (also sow a post, where someone had same % with 39 quests but that might have been an older version oft he mod). Is it a bug or am I missing something? Do I need to unlock all silver skills or build a certain spell/item/structure or kill a certain mob? Can't find a way to unlock the last hidden quest, if there is any :(

Thaumcraft: There are no taintacles at all. I've spent most of the game on my flying platform so 500 blocks around me is nothing but taint yet no taintacle or any taint related mobs (except creepers and poor livestock that wanders into it) can be found. I've read it takes time for it to spawn and also that I have to go further away. So I equipped myself with a lovely vegetarian epic bacon, beloved flügel tiara, sharpy friend Excal and went on an adventure to a land far far away beyond the seven seas. It was way over 3000 blocks away. I've found a Greenland, several templar lairs, couple of friendly soon-to-be-doomed villages and yet, there was no sign of taintacles at all.

TL;DR - taintacles never spawned anywhere no matter the distance or time spent in the area. Still it is possible that I am just an idiot and I'm missing something.

**Comment by ArchieMagi** [comment]
Taintacles: I wonder if it has something to do with the weight spawn? I think you could change that in the Mob Properties Folder in the config file. I did that was tainted swarm, and then proceeded to change it back... Cause tainted swarms make it a bit much for me. ^_^ Also... Can I haz tiara?
